### ISSUE TITLE
code for training supervision transfer initialization weights

### ISSUE THREAD

**Issue by truetqy** [s-gupta/fast-rcnn#2]
Hi, Gupta,
Where to find the code for training supervision transfer initialization weights? 
How did you train ST_vgg_to_alexnet_hha.caffemodel? 
I want to train with different modality. 
Thanks~

**Comment by shinexunju** [comment]
Hi Saurabh,

I'd like to echo @truetqy 's comments. And could you make the carefully selected 10K frames used for supervision transfer public?

Thanks for your sharing.

**Comment by s-gupta** [comment]
Yes, I will make these frames available later this week, sorry for the delay.
